oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java

```diff
@@ -269,6 +274,14 @@ public boolean analyzerConfigSplitOnNumerics() {
         return getOptionalValue(analyzersTree, SPLIT_ON_NUMERICS, false);
     }
 
+    /**
+     * Returns the mapping version for this index definition.
+     * If the version is not specified, the default value is {@code 1.0.0}.
+     */
+    public String getMappingVersion() {
+        return getOptionalValue(definition, PROP_INDEX_MAPPING_VERSION, "1.0.0");
+    }
```

Comment:
Shouldn't the default value be a clearly invalid version, like 0.0.0? Because we will assume that 1.0.0 is a valid version, so it can create confusion in cases where the index is missing a version, we might assume wrongly that it has a version.

Reply:
The mapping version is internal and cannot be changed in the index definition (it's stored in a hidden property). If the index definition does not have a version, we return the base version currently set as `1.0.0`.